lib/light-service/organizer/iterate.rb

```diff
@@ -11,12 +12,39 @@ def self.run(organizer, collection_key, steps)
           item_key = collection_key.to_s.singularize.to_sym
           collection.each do |item|
             ctx[item_key] = item
```

Comment:
I think it could be better to "advance" the item only if we're not inside a rollback.
Also I can't get why we're not `break`ing the `each`'s cycle if we're whilst a _rollback_.

E.g.:

```ruby
            if ctx.key?(:_rollback)
              # Handle Rollback
              rollback_items(
                item,
                organizer,
                ctx,
                collection,
                steps
              )
              break
            else
              ctx[item_key] = item
              ctx = scoped_reduce(organizer, ctx, steps)
            end
```